### DISCUSSION TITLE
重新装机后似乎老版本命令失效了?

### DISCUSSION THREAD

**Discussion by luoY135246** [dashingsoft/pyarmor#1833] [Q&A] [answered]
这是原来下载文档的注册命令  pyarmor register pyarmor-regcode-2462.txt
请问现在注册应该是什么命令

**Comment by jondy** [comment, marked as answer]
请安装 pyarmor == v7.7.4，不要安装 Pyarmor > 8.0，请参考项目主页关于升级的说明

**Comment by luoY135246** [comment]
可以了，不过得回滚到py3.10版本....
